<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labeljpg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>文件标签测试</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonProgressBar">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>测试进度条</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbtnstart">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbtnpause">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbtnreset">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
